templates/angularity/karma.conf.js

```diff
@@ -19,9 +19,13 @@ module.exports = function(config) {
     // list of files to exclude
     exclude: [],
 
+    // register any plugins which  are not siblings of karma in angularity global installation
+    // and thus need to be registered manually
+    plugins: config.plugins.concat(/*ANGULARITY_PLUGIN_LIST*/),
```

Comment:
add to the note some explanation of why you are preserving the existing value, e.g.
`// the existing value will be the auto-detected plugins so we must append to that`